src/App.jsx

```diff
@@ -3,25 +3,16 @@ import './App.css'
 import Layout from './views/Layout/Layout'
 import Home from './views/Home/Home'
 import { fetchUser } from './services/user'
+import { UserProvider } from './Context/UserContext'
 
 function App() {
   // inital value should match the data type of end value
   const [user, setUser] = useState({})
```

Comment:
Since you've moved the state logic out to your custom hook in the `UserProvider`, you don't actually need to call the `useState` hook here anymore. Also, you'll want to clean up unused imports to ensure you're providing clean and maintainable code.